tests/test_operators.py

```diff
@@ -45,23 +46,86 @@ def compare_streams(self, all, expected_all):
             self.assertDictEqual(input_dict, output_dict)
 
     def test_map_instance_values(self):
+        mappers = {"a": {"1": "hi", "2": "bye"}}
+
         inputs = [
-            {"a": 1, "b": 2},
-            {"a": 2, "b": 3},
+            {"a": "1", "b": "2"},
+            {"a": "2", "b": "3"},
         ]
 
         targets = [
-            {"a": "hi", "b": 2},
-            {"a": "bye", "b": 3},
+            {"a": "hi", "b": "2"},
+            {"a": "bye", "b": "3"},
         ]
 
+        # simple value substitute
         check_operator(
-            operator=MapInstanceValues(mappers={"a": {"1": "hi", "2": "bye"}}),
+            operator=MapInstanceValues(mappers=mappers),
             inputs=inputs,
             targets=targets,
             tester=self,
         )
 
+        # process_every_value=True would not accept non-list inputs
+        with self.assertRaises(ValueError):
```

Comment:
Please check exception with the new `check_operator_exception` that also checks the exception text.

    exception_text = "Required filter field ('c') in FilterByValues is not found in {'a': 1, 'b': 2}"
        check_operator_exception(
            operator=FilterByValues(required_values={"c": "5"}),
            inputs=inputs,
            exception_text=exception_text,
            tester=self,
        )